src/models/GameType.ts

```diff
@@ -125,6 +135,7 @@ export function deleteGameTypeById(id: number, transaction?: Transaction) {
     where: {
       id,
     },
+    cascade: true,
     transaction,
   });
```

Comment:
fixing a bug. without the cascade, sequalize would throw error if other tables continue having the gametype ref.

Reply:
ok